internal/graph/cached_resolver_test.go

```diff
@@ -874,6 +933,30 @@ func TestCheckCacheKeyWithContext(t *testing.T) {
 	require.NotEqual(t, key3, key4)
 }
 
+func TestErrorReturnedWhenHigherConsistencyRequested(t *testing.T) {
```

Comment:
I'm not sure i understand this test - if the consistency was `minimize_latency` but the result was not in the cache, we should also return an error, no?

What are you really testing here? It seems like you are testing that `cachedCheckResolver` is delegating to the mock, but don't we already have tests for that?

Reply:
Yeah the consistency setting doesn't have an impact here, I think realistically this is covered by the tests in the test table at the top of the file so can be removed

Reply:
Removed in c3c6d511e255f424d3c85e7a61f9e376cbc5f478